scientist-softserv/webstore
# 360-fix-e2e-specs

# story
fix the broken browse tests **locally**. they will still fail in ci due to configuration. (ref: #378)

Co-authored-by: Adrian Carranza <adriancarranza0113@gmail.com>
Co-authored-by:Jean Aragon <jeanaragon0@gmail.com>
Co-authored-by: Alisha Evans <alishaevn2@gmail.com>

- ref: https://github.com/scientist-softserv/webstore/issues/360

# Screenshots / Video
<img width="1271" alt="image" src="https://github.com/scientist-softserv/webstore/assets/29032869/5513d4f0-bfef-4fe7-a73d-c5e3bbb5e560">

# Testing
1. pull this branch
2. ensure you have all of the environment variables in the `env` property of "cypress.config.js" set up.
3. run `yarn cypress:e2e`
4. choose the "browse" test

# notes
if you see an error like the below, it is not because of the test but of another issue that will be handled separately as it has to do with some dynamic forms not being created properly. (this issue should only affect the beachside marketplace, not phenovista. ref: #376)

<details>
<summary>error</summary>

<img width="686" alt="image" src="https://github.com/scientist-softserv/webstore/assets/29032869/8c7ce79e-2cd0-4b90-b9ed-54eb1dd6507d">
</details>
